### PR TITLE
add inverse transform for plotting

### PR DESCRIPTION
**Pull request by ioannisPApapadopoulos**

### CODE REVIEW

**Comment by ioannisPApapadopoulos** [comment]
@dlfivefifty it passes the tests, ok to merge?

File: src/annulus.jl [hunk 1]
```diff
@@ -179,7 +179,35 @@ function grid(S::ZernikeAnnulus{T}, B::Block{1}) where T
...
+    RadialCoordinate.(r, convert(T,π)*θ')
+end
+
+_angle(rθ::RadialCoordinate) = rθ.θ
+
+function plotgrid(S::ZernikeAnnulus{T}, B::Block{1}) where T
+    N = Int(B) ÷ 2 + 1  # polynomial degree
+    g = grid(S[:,Block.(OneTo(2N))]) # double sampling
+    θ = [map(_angle,g[1,:]); 0]
+    ρ = S.ρ + eps(T)
```
Comment (dlfivefifty) [inline src/annulus.jl:191]: Why `+ eps(T)`? Just to avoid going outside? If so make a comment

**Review by dlfivefifty — APPROVED** [review]

File: src/annulus.jl (hunk 1 above)
Comment (ioannisPApapadopoulos) [reply src/annulus.jl:191]: yes, it sporadically throws an `BoundsError` otherwise. Ok!

**Comment by ioannisPApapadopoulos** [comment]
Could we merge this? @dlfivefifty

**Pull request merged**